src/content/docs/resources/faqs.md

```diff
@@ -38,3 +42,9 @@ Taiko does not have an L2 sequencer, since everyone can become a proposer permis
 ## How do I get ETH on the Taiko Hekla network?
 
 You can use the official [Taiko bridge](https://bridge.hekla.taiko.xyz/) to send your ETH from the Holesky network to the Taiko Hekla network.
+
+# Developer FAQs
+
+## How can I get the L1 block number from L2
+
+You can check the last synced L1 block height in the TaikoL2 contract [here](https://github.com/taikoxyz/taiko-mono/blob/c7f3143c763792806de2dffb09d35e756f4cb1e9/packages/protocol/contracts/L2/TaikoL2.sol#L41).
```

Comment:
maybe link the [hekla contract](https://blockscoutapi.hekla.taiko.xyz/address/0x1670090000000000000000000000000000010001/read-proxy)?